src/v2/view-builder/views/custom-app/ChallengeCustomAppResendPushView.js

```diff
@@ -0,0 +1,43 @@
+import { loc, createCallout } from 'okta';
```

Comment:
This view also is same as `ChallengeOktaVerifyResendPushView`

Reply:
same reason : The strings are all customizable and different for each view. Also we have a different classname.